fili-core/src/main/java/com/yahoo/bard/webservice/web/DataApiRequest.java

```diff
@@ -238,6 +242,18 @@ public DataApiRequest(
 
         this.having = DruidHavingBuilder.buildHavings(this.havings);
 
+
+        if (
+                sorts != null &&
+                        !sorts.isEmpty() &&
+                        sorts.toLowerCase(Locale.ENGLISH).contains(DATE_TIME_STRING.toLowerCase(Locale.ENGLISH))
```

Comment:
Also, why are we lower-casing this keyword? Do we do that anywhere else in the API for keywords?